### PR TITLE
Use updated templeCirculatingSupply

### PR DESCRIPTION
**Pull request by marshall2112**
# Description
What does this PR solve?

Fixes # (issue)

# Checklist
- [ ] Code follows the style guide
- [ ] I have performed a self-review of my own code
- [ ] New and existing tests pass locally
- [ ] This PR is targeting the correct branch

### CODE REVIEW

**Comment by nichosystem** [comment]
Deploy preview for _core_ ready!

✅ Preview
https://core-p0012fmkh-templedao.vercel.app
https://pr-919-preview.templedao.link

Built with commit 312ca214011deae45cad52195ddbdae2e39051a2.
This pull request is being automatically deployed with [vercel-action](https://github.com/marketplace/actions/vercel-action)

File: apps/dapp/src/components/Pages/Core/DappPages/Dashboard/hooks/use-dashboardv2-metrics.ts [hunk 1]
```diff
@@ -272,15 +272,16 @@ export default function useDashboardV2Metrics(dashboardType: DashboardType) {
   };
 
   const getTempleCirculatingSupply = async (): Promise<string> => {
-    const response = await fetchSubgraph<any>(
+    const response = await fetchGenericSubgraph<any>(
+      'https://api.thegraph.com/subgraphs/name/medariox/temple-metrics',
```
[the remark this answers is not in the document]
Comment (marshall2112) [reply apps/dapp/src/components/Pages/Core/DappPages/Dashboard/hooks/use-dashboardv2-metrics.ts:276]: This is a new subgraph, let me check with @medariox and if nothing else will add it to the config properly.

**Review by frontier159 — APPROVED** [review]

File: apps/dapp/src/components/Pages/Core/NewUI/Home.tsx [hunk 2]
```diff
@@ -92,33 +93,33 @@ const Home = ({ tlc }: { tlc?: boolean }) => {
...
+            treasuryPriceIndex
+            templePrice
           }
         }`
       );
       setMetrics({
-        price: parseFloat(ramosData.metrics[0].templePriceUSD),
-        tpi: parseFloat(ramosData.metrics[0].treasuryPriceIndexUSD),
+        price: parseFloat(ramosData.metrics[0].templePrice),
+        tpi: parseFloat(ramosData.metrics[0].treasuryPriceIndex),
```
Comment (medariox) [inline apps/dapp/src/components/Pages/Core/NewUI/Home.tsx:122]: Please use the `templeV2` subgraph for TPI, see [example query](https://api.thegraph.com/subgraphs/name/medariox/v2-mainnet/graphql?query=query+MyQuery+%7B%0A++tpiOracles+%7B%0A++++currentTpi%0A++%7D%0A%7D).

File: apps/dapp/src/components/Pages/Core/NewUI/Home.tsx [hunk 3]
```diff
@@ -92,33 +93,33 @@ const Home = ({ tlc }: { tlc?: boolean }) => {
...
-            templePriceUSD
+            treasuryPriceIndex
+            templePrice
           }
         }`
       );
       setMetrics({
-        price: parseFloat(ramosData.metrics[0].templePriceUSD),
-        tpi: parseFloat(ramosData.metrics[0].treasuryPriceIndexUSD),
+        price: parseFloat(ramosData.metrics[0].templePrice),
```
Comment (medariox) [inline apps/dapp/src/components/Pages/Core/NewUI/Home.tsx:121]: Please use the `ramos` subgraph for the price, see [example query](https://api.thegraph.com/subgraphs/name/templedao/templedao-ramos/graphql?query=query+MyQuery+%7B%0A++metrics+%7B%0A++++spotPrice%0A++%7D%0A%7D).

Comment (marshall2112) [reply apps/dapp/src/components/Pages/Core/NewUI/Home.tsx]: Done https://github.com/TempleDAO/temple/pull/919/commits/4d016313dc0708a24fd0946725faa519ad1b49b6

File: apps/dapp/src/components/Pages/Core/NewUI/Home.tsx (hunk 2 above)
Comment (marshall2112) [reply apps/dapp/src/components/Pages/Core/NewUI/Home.tsx]: Done https://github.com/TempleDAO/temple/pull/919/commits/4d016313dc0708a24fd0946725faa519ad1b49b6

**Review by medariox — APPROVED** [review]
LGTM 👍

**Pull request merged**